image_retrieval/dataloaders/data_module.py

```diff
@@ -62,14 +65,14 @@ def train_dataloader(self):
         return DataLoader(
             self.train_dataset,
             batch_size=self.train_batch_size,
-            collate_fn=collate_fn,
+            # collate_fn=collate_fn,
```

Comment:
Why are we removing `the collate_fn`?

Reply:
Didn't remove the `collate_fn`, was just trying to benchmark the training time with and without it.